R/lat_bins.R

```diff
@@ -20,6 +20,8 @@
 #' @param plot \code{logical}. Should a plot of the latitudinal bins be
 #' generated?
 #' @return A \code{dataframe} of latitudinal bins of user-defined size.
+#' @seealso
+#' For equal-area latitudinal bins, see \link{lat_bins_equal}.
```

Comment:
I wonder whether we could make the distinction between the two functions more clear in the names. Really `lat_bins()` is equal by degrees and `lat_bins_equal()` is equal by area - would `lat_bins_degrees()` and `lat_bins_area()` be better? Maybe you have a better idea.

Reply:
I completely agree with you... I was just in two minds about how we treat the current `lat_bins()`... but for now, I've gone with:

- Updating `lat_bins_equal()` to `lat_bins_area()`
- Adding a `lat_bins_degrees()` alias for `lat_bins()`  
- Added a lifecycle badge and note for `lat_bins()` to say that the function has been deprecated and will be removed in the future

What do you think @bethany-j-allen - happy with those changes? @KEichenseer any additional thoughts here?

Reply:
This seems like a good solution to me!